tests/compute/test_nodeflow.py

```diff
@@ -98,7 +98,7 @@ def check_basic(g, nf):
     for i in range(1, nf.num_layers):
         in_deg = nf.layer_in_degree(i)
         out_deg = nf.layer_out_degree(i - 1)
-        assert F.asnumpy(F.sum(in_deg, 0) == F.sum(out_deg, 0))
+        assert F.asnumpy(in_deg).sum(0) == F.asnumpy(out_deg).sum(0)
```

Comment:
It seesm these two impls are equivalent, why converting to numpy first?

Reply:
Chainer variables don't support things like `a == b`.

Reply:
I see...